docs/how-tos/nexus-validation-dev-notes.md

```diff
@@ -0,0 +1,473 @@
+# NeXus Validation Dev. Notes
+
+# 1\. Overview of validation methods
+
+nexusformat.org listed two programs for the verification and validataion of NeXus files:  
+[https://manual.nexusformat.org/validation.htm](<https://manual.nexusformat.org/validation.html>)
+
+1. nxvalidate
+
+2. punx
+
+## nxvalidate
+
+[https://github.com/nexusformat/cnxvalidate](<https://github.com/nexusformat/cnxvalidate>)
+
+### Pro:
+
+1. FAIRmat and NIAC NeXus definiton can be used.
+
+2. Easy to install with Linux
+
+3. Points out some Errors with pynxtools-ellips created file - our fault?
+
+### Con: 
+
+1. Is C/C++ based
```

Comment:
How is this a con? Tons of software in these languages.

Reply:
We are not used so much to C. So we would have difficulties/limitations if we would like to fix/extend a few features.

Reply:
Ok, if this is an internal document, that's fine.

I think we can still bring a lot of that stuff to the official docs, just need to remove these opinions that don't help external people too much.

Reply:
Okay good, then I can include it that way. Ill remove the opinions and keep the rest.